commands/system_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestGetBusyCmd() {
+	s.Run("GetBusy when not set", func() {
+		printer.Clean()
+		sbs := &model.ServerBusyState{}
+
+		s.client.
+			EXPECT().
+			GetServerBusy().
+			Return(sbs, &model.Response{Error: nil}).
+			Times(1)
+
+		err := getBusyCmdF(s.client, &cobra.Command{}, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], sbs)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("GetBusy when set", func() {
+		printer.Clean()
+		const minutes = 15
+		expires := time.Now().Add(time.Minute * minutes).Unix()
+		sbs := &model.ServerBusyState{Busy: true, Expires: expires}
+
+		s.client.
+			EXPECT().
+			GetServerBusy().
+			Return(sbs, &model.Response{Error: nil}).
+			Times(1)
+
+		err := getBusyCmdF(s.client, &cobra.Command{}, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], sbs)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("GetBusy with error", func() {
+		printer.Clean()
+		s.client.
+			EXPECT().
+			GetServerBusy().
+			Return(nil, &model.Response{Error: &model.AppError{Id: "Mock Error"}}).
+			Times(1)
+
+		err := getBusyCmdF(s.client, &cobra.Command{}, []string{})
+		s.Require().NotNil(err)
```

Comment:
I think we are preferring Require().Error(err) for these cases.

Reply:
Fixed